Library/Formula/rust.rb

```diff
@@ -23,6 +30,26 @@ def install
     system "./configure", *args
     system "make"
     system "make install"
+
+    resource("cargo").stage do
+      # as soon as cargo can be built without gated features,
+      # running ./travis.install.deps.sh should be useless
+      # and --local-rust-root should be able to point to `prefix`
+
+      args = ["--prefix=#{prefix}"]
+      if build.head?
+        args << "--local-rust-root=\"#{prefix}\""
+      else
+        system "./.travis.install.deps.sh"
+        args << "--local-rust-root=\"$PWD\"/rustc"
```

Comment:
Wasn't sure how to get the path of a staged resource (which is what `$PWD` is equal to here). Is that possible?